main.py

```diff
@@ -1,10 +1,59 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json") as file:
+        json_data = json.load(file)
+
+    for player in json_data:
+        player_info = json_data[player]
+        if Race.objects.filter(
```

Comment:
use  `get_or_create` instead of this code here and other places when you must get or create an object . https://www.queworx.com/django/django-get_or_create/.